fdk/context.py

```diff
@@ -19,25 +19,15 @@
 from fdk import headers as hs
 
 
-def set_response_headers(current_headers, new_headers,
+def set_response_headers(current_headers, new_headers: dict,
                          status_code, content_type=None):
-    if isinstance(new_headers, dict):
-        new_headers = hs.GoLikeHeaders(new_headers)
-    elif isinstance(new_headers, hs.GoLikeHeaders):
-        pass
-    else:
-        raise TypeError(
-            "Invalid headers type: {}, only dict allowed."
-            .format(type(new_headers))
-        )
-
```

Comment:
is removing go like headers going to cause a breakage? that might be okay, just would like some list of these things being broken if we're breaking things (and some expectations, as noted on other PR)

Reply:
@denismakogon if there's less breakages here, then it may be worth making this patch independent of #62 so that we can land it, there's a bit of time pressure to resolve the issue around aiohttp taking too long to load and we may be able to be more patient with #62 since it is an orthogonal improvement... let me know what you'd like to do, i'm happy to help rebase things out too...

Reply:
Nothing breaks here. Go-like headers was around because of unusual type of headers in json/CloudEvents format, now it’s not necessary and should disappear. No need it backward compatibility.

Moreover, go-like headers was only for an internal use, not for the customers.